engines/auth_by/app/controllers/auth_by/ext/core_by/base/api_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module AuthBy
+  module Ext
+    module CoreBy
+      module Base
+        module ApiController
+          extend ActiveSupport::Concern
+
+          included do
+            # Support session-cookie-based authentication
+            include ActionController::Cookies
+
+            include Authenticatable
+          end
```

Comment:
With `ActiveSupport::Concern`, we can include dependencies right into the module without the `included` hook.